@xen-orchestra/web-core/lib/components/card/CardTitle.vue

```diff
@@ -0,0 +1,64 @@
+<!-- v1.0 -->
+<template>
+  <div class="card-title">
+    <div class="left-blok">
+      <div class="title-and-counter">
+        <span class="typo h6-medium"><slot name="left"></slot></span>
```

Comment:
As this is the main slot, it should be the _default_ slot:
```suggestion
        <span class="typo h6-medium"><slot /></span>
```